src/FBAppEventsLogger.ts

```diff
@@ -134,13 +134,7 @@ export type AppEventParam = {
   ValueYes: string;
 };
 
-let AppEvents: AppEvent | undefined = undefined;
-let AppEventParams: AppEventParam | undefined = undefined;
-if (AppEventsLogger !== undefined) {
-  const constants = AppEventsLogger?.getConstants();
-  AppEvents = constants.AppEvents;
-  AppEventParams = constants.AppEventParams;
-}
+const {AppEvents, AppEventParams} = AppEventsLogger?.getConstants() || {};
```

Comment:
This is not a firm suggestion, just a typescript thought - would nullish coalescing be better here?

```suggestion
const {AppEvents, AppEventParams} = AppEventsLogger?.getConstants() ?? {};
```

Reply:
It depends, in this circumstance  I'm using `||` for a reason, in the image below you can see `||` generates lesser js code than `??`, so usually in my work we use `||` instead of `??`. 

<img width="974" alt="example" src="https://user-images.githubusercontent.com/48589760/150841549-531a9e3d-4565-4790-bdbb-834d4de69d95.png">

But `??` is still useful in some scenarios, like if the return value is falsely

```
// data[key] will probably be 0 but we only want defaultValue if it's undefined
data[key] ?? defaultValue
```

Reply:
works for me :-), thanks for the explanation